oracle/Forc.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "oracle"
+
+[constants]
+owner = { type = "b256", value = "0x09c0b2d1a486c439a87bcba6b46a7a1a23f3897cc83a94521a96da5c23bc58db" }
```

Comment:
Where did this hardcoded value come from?

Reply:
It's the first wallet address generated deterministically by the SDK

Reply:
The deterministic wallets is a new feature not included in the latest version of fuels.  If you check out my Cargo.toml you'll see I'm getting my fuels from github.

Reply:
Do these changes also include the ".DS_Store" binary file?

Reply:
Yeah it looks like it in files changed.  However, I don't see the file locally...  I could add it to the .gitignore but honestly I don't really know what it is

Reply:
Weird I see it with ls -a I guess I can just delete it?  I assume we don't want this file

Reply:
Any file that does not have a name that starts with `.` for example `.gitignore` will not show using ls unless you give the all command. Are you using a Mac?

Reply:
A quick google search is telling me it is a macOS file.